src/common/components/ui/Notification/index.tsx

```diff
@@ -35,6 +35,7 @@ const Notification: React.FC = () => {
       }}
     >
       {
+        // eslint-disable-next-line security/detect-object-injection
```

Comment:
Please avoid using inline eslint rules, it is better to use global rules, and if there is an exception, we should refactor whenever possible, or, as a last resource, create a new global rule disabling the entire rule.